.prettierrc.json

```diff
@@ -3,5 +3,13 @@
   "bracketSameLine": true,
   "printWidth": 100,
   "singleQuote": true,
-  "arrowParens": "avoid"
+  "arrowParens": "avoid",
+  "overrides": [
+    {
+      "files": [".mocharc.jsonc"],
+      "options": {
+        "trailingComma": "none"
```

Comment:
are they supported? if yes we can enable trailing commas

Reply:
No, mocha supports JSONC but will throw an error if the file has trailing commas.

Reply:
ok!